base/doc/usrguide.tex

```diff
@@ -794,6 +794,59 @@ \subsection{Commands at the start of tabular cells}
 \end{tabular}
 \end{verbatim}
 
+\subsection{Using the verbatim argument types}
+
+As described above, the \texttt{v}-type argument may be viewed as similar to
+\cs{verb}. Before looking at exactly what that means, it is important to
+highlight some key differences. Most notably, \emph{grabbing} a verbatim-like
+argument is separate from \emph{typesetting} it: the latter is covered in the
+next section.
+
+When grabbing a \texttt{v}-type argument, \LaTeX{} first uses the kernel
+command \cs{dospecials} to turn off the \enquote{special} nature of characters.
+It then makes both spaces and tabs \enquote{active}, so they can be given a
+custom definition. Any other characters are grabbed as-is: this means that if
+any characters have been made \enquote{special} and are not listed in
+\cs{dospecials}, an error will arise (see below).
+
+Characters are grabbed two identical delimiters: in contrast to \cs{verb}, the
+characters \texttt{\textbackslash}, |{|, |}| and |%| \emph{cannot be used}. If
+any of the grabbed tokens have \enquote{special} meaning, an error will be
+issued.
+
+For the \texttt{+v}-type argument, which allows line breaks in input,
+newline characters are converted into \cs{obeyedline} commands. This means that
+the grabbed tokens can be used directly in typesetting, while a local
+redefinition of \cs{obeyedline} can be used to achieve other outputs. For
+example, to retain blank lines whilst typesetting, one could use
+\begin{verbatim}
+\renewcommand*\obeyedline{\mbox{}\par}
+\end{verbatim}
+along with other setting that may be appropriate (e.g., using a monospaced
+font).
+
+Some additional details that may be useful for those with more \TeX{}
+knowledge: do not worry if this does not make sense to you! Grabbed tokens are
+all stored as either active or \enquote{other}: spaces, tabs and non-ASCII
+codepoints in $8$-bit engines are active, everything else is detokenized. As
```

Comment:
I cannot follow the logic here!

There probably should be a phrase such as:
when using an 8-bit engine, . . . , but otherwise . . . 

And do. to use both “catcode/type other” and “detokenized” so close to each other, please! 

I am unsure if the last sentence is useful! Especially if starting “as such”?